pom.xml

```diff
@@ -109,6 +109,11 @@
 			<version>${mockito.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-inline</artifactId>
+			<version>${mockito.version}</version>
+		</dependency>
```

Comment:
Isn't the MockMaker included in Mockito 4.x?

Reply:
Yes, just removed the [org.mockito.plugins.MockMaker](https://github.com/cryptomator/fuse-nio-adapter/pull/66/commits/3edcab74dce506dcb3c6065468bd09213e1b63a9#diff-b8f025a6bc4fe418ee263e43aeb2678327f367a433bd2792d191f7a7f893775b) file, thanks for mentioning. `mockito-inline` is still required, otherwise the new tests fails with `Mockito can only mock non-private & non-final classes.`.

Reply:
strange, I can't recall adding this dependency in any other repo, such as cryptofs, despite heavily mocking basically everything